cra-config.yaml

```diff
@@ -4,6 +4,8 @@ CRA_TARGETS:
   - CRA_TARGET: "examples/complete" # Target directory for CRA scan. If not provided, the CRA Scan will not be run.
     CRA_IGNORE_RULES_FILE: "cra-tf-validate-ignore-rules.json" # CRA Ignore file to use. If not provided, it checks the repo root directory for `cra-tf-validate-ignore-rules.json`
     PROFILE_ID: "0e6e7b5a-817d-4344-ab6f-e5d7a9c49520" # SCC profile ID (currently set to the FSCloud 1.4.0 profile).
+    TF_VAR_regions: "us-south"
+    TF_VAR_prefix: "complete-kms"
```

Comment:
these are not in correct place in yaml. They need to be under `CRA_ENVIRONMENT_VARIABLES`. See the commented out code below this

Reply:
@ocofaigh fixed.